### PR TITLE
2020 syllabus updates - final checks

### PR DESCRIPTION
**Pull request by lachlandeer**
This PR prepares us for the Winter 2020 version by doing the following:

* updates course meta-data, #2  
* updates the schedule #3 
* distinguishes requirements for 2nd year vs 1st year students #4 
    - Deliberately vague in preliminary versions until we get official confirmation from department
* slightly updates motivation, #6 

I also add a `snakemake` workflow that automates the building the syllabus thru an Rmd + knitr setup

Comments + request for changes welcomed (made changes even better)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 *   Module Maintainer: Lachlan Deer (`@lachlandeer`)
 *   Course: [Programming Practices for Research in Economics](https://github.com/pp4rs)
 *   Institute: Dept of Economics, Uni Zurich
-*   Current Version: [Fall 2017 edition](https://pp4rs.github.io/2017-uzh)
+*   Current Version: [Winter 2020 edition](https://pp4rs.github.io/2020-uzh)
```
Comment (bergmul) [inline README.md:8]: Website link not live yet.

**Review by bergmul — APPROVED** [review]
I added my suggestions to the file. Have a look if you like them before you merge.

* General update of first motivation (audience is still in there and in audience section - could be unified but it's sufficient)
* I added a page brake to have the schedule on a single page (now it looks weird on the other pages. could add page break after motivation)
* Added stuff to the tools where I felt like it.

**Comment by bergmul** [comment]
For future generations, we should also add a dependency check for the "here" and "Rmarkdown" R packages. Currently, snakemake does give no indication that this might be a problem if it doesn't compile. I requires the user to load the Rmd file into Rstudio and to replace all input readers with hardcoded equivalents to see that this causes the error.

**Comment by lachlandeer** [comment]
I'm fine with the changes ... merge it when you feel like it :D

**Comment by bergmul** [comment]
done.

**Pull request merged**